include/libtorrent/announce_entry.hpp

```diff
@@ -128,12 +130,8 @@ TORRENT_VERSION_NAMESPACE_2
 	// this class holds information about one listen socket for one tracker
 	struct TORRENT_EXPORT announce_endpoint
 	{
-		friend class torrent;
-#if TORRENT_ABI_VERSION == 1
+		friend class lt::torrent;
```

Comment:
now that `announce_endpoint` also is in the versioned namespace, the reference to `announce_entry` is unambiguous. However, `torrent` needs to be fully qualified